### PR TITLE
Ocp set ieee1667

### PR DESCRIPTION
**Pull request by ikegami-t**

### CODE REVIEW

File: plugins/ocp/ocp-nvme.c [hunk 1]
```diff
@@ -2928,6 +2927,70 @@ static int get_enable_ieee1667_silo(int argc, char **argv, struct command *cmd,
...
+	struct ieee1667_get_cq_entry cq_entry;
+	int err;
+	const __u8 fid = 0xc4;
+
+	struct nvme_set_features_args args = {
+		.result = (__u32 *)&cq_entry,
+		.args_size = sizeof(args),
+		.fd = dev_fd(dev),
+		.timeout = NVME_DEFAULT_IOCTL_TIMEOUT,
+		.cdw11 = OCP_SET(argconfig_parse_seen(opts, "enable"), ENABLE_IEEE1667_SILO),
```
Comment (igaw) [inline plugins/ocp/ocp-nvme.c:2942]: Please use a variable for the `OCP(...)` part. This is a bit too much in one line.

File: plugins/ocp/ocp-nvme.h [hunk 2]
```diff
@@ -14,6 +14,8 @@
 #define OCP_PLUGIN_VERSION   "2.9.3"
 #include "cmd.h"
 
+#define ENABLE_IEEE1667_SILO "enable IEEE1667 silo"
```
Comment (igaw) [inline plugins/ocp/ocp-nvme.h:17]: I would use the string directly in the command list macro. Just to keep it consistent with the rest.

**Comment by ikegami-t** [comment]
Just fixed for the review comments. Thank you.

**Comment by igaw** [comment]
Thanks!

**Pull request merged**